modules/040-control-plane-manager/openapi/config-values.yaml

```diff
@@ -44,6 +44,12 @@ properties:
               A list of API audiences to add when provisioning ServiceAccount tokens.
             items:
               type: string
+          additionalAPIIssuers:
+            type: array
+            description: |
+              A list of API issuers to add when provisioning ServiceAccount tokens.
```

Comment:
The description has to be detailed. Let's add at least what is in the Kubernetes docs.

> A list of API issuers used to determine which issuers are accepted. This option is beneficial when migrating from one issuer to another.

Reply:
https://github.com/deckhouse/deckhouse/pull/9739/commits/52761c39746a769fd9291f9ccaf26ade54818c2b